docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -81,18 +81,27 @@ Server ID                            Hostname        Services Version Upgrader
 
 </Details>
 
-<Tabs dropdownCaption="Cluster Type" dropdownSelected="Self-hosted">
-<TabItem label="systemd" options="Self-hosted">
+<Tabs dropdownView dropdownCaption="Cluster Type">
+<TabItem label="Self-hosted" scope="enterprise">
+<Tabs>
+<TabItem label="systemd" scope="enterprise">
 
-Confirm you have the Teleport Enterprise edition installed.
+### Step 1/3. Ensure the Teleport repository is added and Teleport Enterprise is installed
 
+To check if teleport was installed using the package manager:
 ```code
-$ teleport version
-Teleport Enterprise v(=teleport.version=) go(=teleport.golang=)
+$ sudo apt list teleport-ent
```

Comment:
Will this show anything if the .deb or .rpm package was installed directly from a file (`dpkg -i`/`rpm -i`) rather than from a repository? Cause in this case they'd also need to add the repo source.

Reply:
Good point. How about using `apt-cache policy` or `yum repolist`?

Reply:
@fheinecke What would be a good way to reliably tell if a teleport package was installed from a repository or a file?

Reply:
@r0mant That depends - do you want to know if _any_ Teleport package has been installed on the system via a repository? Or do you want to know if the `tsh` command corresponds to a Teleport package installed on the system via a repository? Or something else?

As far as I am aware there is no 100% perfect way to do either, but there are a few ways to do each that are probably (hopefully) good enough for this application.

Reply:
The former - to let users know whether they need to add our repository to install the updater.

Reply:
What about just checking if `/etc/apt/sources.list.d/teleport.list` or `/etc/yum.repos.d/teleport.repo` exists?

Reply:
@bernardjkim Sounds good to me, this is what our install instructions recommend.